libs/remix-tests/src/run.ts

```diff
@@ -115,7 +115,7 @@ commander
     const provider: any = new Provider()
     await provider.init()
     web3.setProvider(provider)
-
+    extend(web3)
```

Comment:
why do you need to extend web3?

Reply:
to use `eth_getHHLogsForTx`

Reply:
from https://github.com/ethereum/remix-project/blob/master/libs/remix-simulator/src/provider.ts#L106